2021q3/opensearch.adoc

```diff
@@ -0,0 +1,15 @@
+== OpenSearch on FreeBSD
+
+Links: +
+link:https://opensearch.org/[OpenSearch website] URL: link:https://opensearch.org/[https://opensearch.org/] +
+link:https://discord.gg/NmtQGAWY[OpenSearch Community (Unoffical) on Discord] URL: link:https://discord.gg/NmtQGAWY[link:https://discord.gg/NmtQGAWY]
```

Comment:
s/Unoffical/Unofficial/ (or maybe a lowercase u, not sure)

Reply:
Fixed, thanks!